docs/mindsdb-docs/docs/sql/tutorials/diabetes.md

```diff
@@ -0,0 +1,290 @@
+
+*Dataset:[Diabetes Data](https://github.com/mindsdb/mindsdb-examples/blob/a43f66f0250c460c0c4a0793baa941307b09c9f2/others/diabetes_example/dataset/diabetes-train.csv)*
+
+Diabetes is a metabolic disease that causes high blood sugar and if left untreated can damage your nerves, eyes, kidneys and other organs. It is known as a silent killer, as recent studies have shown that by the year 2040 the world's diabetic patients will reach 642 million. The need to analyse vast medical data to assist in the diagnoses, treatment and management of illnesses is increasing for the medical community. With the rapid development of machine learning, it has been applied to many aspects of medical health and is transforming the health care system.
+
+The vitality to intelligently transform information into valuable knowledge through machine learning has become more present in biosciences. With the use of predictive models, MindsDB can assist in classifying diabetic and non-diabetic patients or those who pose a high risk. This is just a small showcase on how MindsDB's machine learning will be able to assist in vastly enhancing the reach of illnesses, thereby making it more efficient and can revolutionize businesses and most importantly the health care system.
+
+In this tutorial we will be exploring how we can use a machine learning model to classify negative and positive cases for diabetes.
+**MindsDB** allows you to train your model from CSV data directly, however for this tutorial you will:
+
+1. Create & setup a new database.
+2. Load sample data into a table using the PGAdmin GUI Tool.
+3. Allow connections to the database using Ngrok.
+4. Setup a MindsDB connection to your local instance.
+5. Create a predictor using SQL.
+
+## Pre-requisites
+
+For this tutorial, Docker is highly recommended. A `docker-compose.yml` file will be provided to get you started quickly.
+
+To ensure you can complete all the steps, make sure you have access to the following tools:
+
+1. A MindsDB instance. Check out the installation guide for [Docker](https://docs.mindsdb.com/deployment/docker/) or [PyPi](https://docs.mindsdb.com/deployment/pypi/). You can also use [MindsDB Cloud](https://docs.mindsdb.com/deployment/cloud/).
+2. A PostgreSQL Database. You can install it [locally](https://www.postgresql.org/download/) or through [Docker](https://hub.docker.com/_/postgres).
+3. Downloaded the dataset. You can get it from [here](https://github.com/mindsdb/mindsdb-examples/blob/a43f66f0250c460c0c4a0793baa941307b09c9f2/others/diabetes_example/dataset/diabetes-train.csv)
+4. Access to PGAdmin4 (provided with the docker-compose file).
+5. Optional: Access to ngrok. You can check the installation details at the [ngrok website](https://ngrok.com/).
+
+## Docker
+
+1. Create a new project directory named e.g. `mindsdb-tutorial`
+2. Inside your project directory:
+2.1 Create a new file named `docker-compose.yml`.
```

Comment:
Add indentation here so they can be visible as sub items

Reply:
amended*